Doc/library/subprocess.rst

```diff
@@ -483,6 +483,12 @@ functions.
       The *start_new_session* parameter can take the place of a previously
       common use of *preexec_fn* to call os.setsid() in the child.
 
+   .. versionchanged:: 3.8
+
+      The *preexec_fn* parameter is no longer supported in subinterpreters.
+      The new restriction may affect applications that are deployed in
+      mod_wsgi, uWSGI, and other environments.
```

Comment:
I'm not sure that it's useful to document affected applications.

Reply:
The list of affected applications makes it easier for affected users to find the culprit. In my experience it's mostly embedded WSGI servers and embedded applications that use subinterpreters. mod_wsgi and uWSGI are the most common ones.

Reply:
@gpshead The subinterpreter and whatsnew entries now mention RuntimeError.